tutorials/export/exporting_for_android.rst

```diff
@@ -29,8 +29,8 @@ Download and install the Android SDK.
   - Ensure that the `required packages <https://developer.android.com/studio/intro/update#recommended>`__ are installed as well.
 
     - Android SDK Platform-Tools version 30.0.5 or later
-    - Android SDK Build-Tools version 30.0.3
-    - Android SDK Platform 31
+    - Android SDK Build-Tools version 32.0.0
+    - Android SDK Platform 32
```

Comment:
@garryod Here and below, can you bump the build-tools version to `33.0.2` and the SDK platform version to `33`.

This will match the new versions in https://github.com/godotengine/godot/pull/75203

@mhilbrunner We should wait until https://github.com/godotengine/godot/pull/75203 is submitted to merge this update.